src/indicator/algorithm/bollingerBands.js

```diff
@@ -19,8 +19,8 @@ export default function() {
         return d.bollingerBands;
     };
 
-    d3.rebind(bollingerBands, mergedAlgorithm, 'merge');
-    d3.rebind(bollingerBands, bollingerAlgorithm, 'windowSize', 'value', 'multiplier');
+    rebind(bollingerBands, mergedAlgorithm, 'merge');
```

Comment:
Let's remove it when we split out the algorithms and renderers, that'll solve the visual test problem as well.